src/app/js/admin/removedResources/ImportButton.js

```diff
@@ -0,0 +1,78 @@
+import React, { useEffect, useState } from 'react';
+import { compose } from 'recompose';
+import { polyglot as polyglotPropTypes } from '../../propTypes';
+import translate from 'redux-polyglot/translate';
+import UploadIcon from '@mui/icons-material/Upload';
+import { Button, CircularProgress, styled } from '@mui/material';
+import { importHiddenResources } from '../api/hiddenResource';
+import { toast } from '../../../../common/tools/toast';
+import { useLocation, Redirect } from 'react-router-dom';
+
+const VisuallyHiddenInput = styled('input')({
+    clip: 'rect(0 0 0 0)',
+    clipPath: 'inset(50%)',
+    height: 1,
+    overflow: 'hidden',
+    position: 'absolute',
+    bottom: 0,
+    left: 0,
+    whiteSpace: 'nowrap',
+    width: 1,
+});
+
+const ImportButton = ({ p: polyglot }) => {
+    const buttonLabel = polyglot.t('import');
+    const location = useLocation();
+    const [uploading, setUploading] = useState(false);
+    const [done, setDone] = useState(false);
+
+    const handleFileChange = async event => {
+        setUploading(true);
+        const file = event.target.files[0];
+        const formData = new FormData();
+        formData.append('file', file);
+        const response = await importHiddenResources(formData);
+        setUploading(false);
+        if (response.error) {
+            toast(polyglot.t('import_error'), {
+                type: toast.TYPE.ERROR,
+            });
+        } else {
+            setDone(true);
+            toast(polyglot.t('import_successfull'), {
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
**nitpick (non-blocking):** ‏

```suggestion
            toast(polyglot.t('import_successful'), {
```